### PR TITLE
supplement the configuration of haproxy

### PR DESCRIPTION
**Pull request by allenleu**
add Haproxy configuration items to the document

### CODE REVIEW

**Review by fisherxu — COMMENTED** [review]
Thanks for the enhancement, and do we also need to make haproxy highly available using like  keepalived?

**Comment by allenleu** [comment]
> Thanks for the enhancement, and do we also need to make haproxy highly available using like keepalived?

Yes, we still need to use Keepalived.

**Comment by zhu733756** [comment]
It seems a static configuration example.   Can it be generated with a script? Will it also support a profile for the newer keadm? /cc @fisherxu

File: build/cloud/ha/README.md [hunk 1]
```diff
@@ -158,3 +158,33 @@ fi
 
 ```
 
+## haproxy
```
Comment (fisherxu) [inline build/cloud/ha/README.md:161]: I think the haproxy need to config with keeppalived together?

**Comment by fisherxu** [comment]
> It seems a static configuration example. Can it be generated with a script? Will it also support a profile for the newly keadm? /cc @fisherxu

For ha deploy, we can provide the static docs. In future we can also support ha deploy in keadm? @zhu733756

**Comment by zhu733756** [comment]
> > It seems a static configuration example. Can it be generated with a script? Will it also support a profile for the newly keadm? /cc @fisherxu
> 
> For ha deploy, we can provide the static docs. In future we can also support ha deploy in keadm? @zhu733756

Maybe it can work by giving a list of ha IPs to do dynamically configure.  What do you think? @fisherxu @allenleu

**Comment by allenleu** [comment]
> > > It seems a static configuration example. Can it be generated with a script? Will it also support a profile for the newly keadm? /cc @fisherxu
> > 
> > 
> > For ha deploy, we can provide the static docs. In future we can also support ha deploy in keadm? @zhu733756
> 
> Maybe it can work by giving a list of ha IPs to do dynamically configure. Cloudcore IPs can get by kubeclient. What do you think? @fisherxu @allenleu

Yes, maybe we can dynamically configure it. But now we add an example to the document, is it easier for users to understand?

File: build/cloud/ha/README.md (hunk 1 above)
Comment (allenleu) [reply build/cloud/ha/README.md:161]: > I think the haproxy need to config with keeppalived together?

Yes, I can sort out the configuration information of keepalived.

**Pull request closed without merging**